preview/MsixCore/MsixCoreInstaller/FileTypeAssociation.cpp

```diff
@@ -335,7 +320,7 @@ HRESULT FileTypeAssociation::ProcessFtaForRemove(Fta& fta)
     for (auto extensionName = fta.extensions.begin(); extensionName != fta.extensions.end(); ++extensionName)
     {
         bool registryHasExtension = false;
-        RETURN_IF_FAILED(m_registryDevirtualizer->HasFTA(*extensionName, registryHasExtension));
+        RETURN_IF_FAILED(m_msixRequest->GetRegistryDevirtualizer()->HasFTA(*extensionName, registryHasExtension));

```

Comment:
>RETURN_IF_FAILED(m_msixRequest->GetRegistryDevirtualizer()->HasFTA(*extensionName, registryHasExtension)); [](start = 8, length = 106)

because ProcessFtaForAdd always adds whatever's in the manifest, we shouldn't filter these out on uninstall.

Reply:
Removed call to HasFTA method, modified ProcessFTAForRemove so that all extensions are removed followed by progid.